### ISSUE TITLE
Job reports as "Failed" if PR already exists

### ISSUE THREAD

**Issue by Mitchell-Eaton** [fjogeleit/yaml-update-action#554]
Using this action with
```
...
          commitChange: true
          branch: mergeThisToUpdateDev
          createPR: 'true'
...
```
"Fails" if the PR already exists:
```
HttpError: Validation Failed: 
{"resource":"PullRequest","code":"custom","message":
"A pull request already exists for foo:mergeThisToUpdateDev."}
```
However, the job is doing exactly what I'd expect, updating the yaml file and committing the new change to the branch after each run.
![image](https://github.com/fjogeleit/yaml-update-action/assets/62262680/c664dba3-0496-4648-b175-9dd46cab82da)

In my use case, "createPR" would work best if it didn't report failure on the PR already existing (or if there was an additional option for that). The end-state is still such that a PR exists.

**Comment by fjogeleit** [comment]
Sorry for the late response, I’ll take a look but it can take a bit.

**Comment by deleze** [comment]
I have the same issue.

**Comment by fjogeleit** [comment]
I merged a change which should only add an info logging if a PR already exists. Its available in `main`. Could you confirm that it solves your issue? Then I create a release for it.

**Comment by Mitchell-Eaton** [comment]
Just now, Jan 30, my GHA still "failed".
```
Run fjogeleit/yaml-update-action@main
  with:
    commitChange: true
    branch: mergeThisToUpdateDev
    createPR: true
    message: Update Dev Image Version to 0.1.62
    changes: {
    "helm/dev-values.yaml": {
      "image.tag": "0.1.62"
    }
  }
  ```
  `HttpError: Validation Failed: {"resource":"PullRequest","code":"custom","message":"A pull request already exists for foo:mergeThisToUpdateDev."}`

**Comment by fjogeleit** [comment]
Okay, I’ll take a second look

**Comment by fjogeleit** [comment]
@Mitchell-Eaton can you please check again? Updated some tooling, for any reason where my changes not reflected in the builded code. Should be fine now. Tested it by myself.

**Comment by Mitchell-Eaton** [comment]
Great - I can see my GHA's running green and simply printing "Pull Request already exists" as info.
Thanks!

**Comment by fjogeleit** [comment]
Released as [v0.14.0](https://github.com/fjogeleit/yaml-update-action/releases/tag/v0.14.0)
